src/test/java/fi/metatavu/metaform/server/test/functional/tests/ReplyTestsIT.kt

```diff
@@ -704,26 +704,21 @@ class ReplyTestsIT : AbstractTest() {
     fun testReplyPagination() {
         TestBuilder().use { testBuilder ->
             val metaform = testBuilder.systemAdmin.metaforms.createFromJsonFile("simple")
-            // dont need to save the replies to a variable
             testBuilder.test1.replies.createSimpleReply(metaform.id!!, "pagination-test-1", ReplyMode.CUMULATIVE)
             testBuilder.test1.replies.createSimpleReply(metaform.id, "pagination-test-2", ReplyMode.CUMULATIVE)
             testBuilder.test1.replies.createSimpleReply(metaform.id, "pagination-test-3", ReplyMode.CUMULATIVE)
-
             val allReplies = testBuilder.systemAdmin.replies.listReplies(metaform.id, null, null, null, null, null, null, null, null, null, null, null)
             assertEquals(allReplies.size, 3)
             assertEquals(allReplies[0].data!!["text"], "pagination-test-1")
             assertEquals(allReplies[1].data!!["text"], "pagination-test-2")
             assertEquals(allReplies[2].data!!["text"], "pagination-test-3")
-
-            val replies1To2 = testBuilder.systemAdmin.replies.listReplies(metaform.id, null, null, null, null, null, null, null, null, 2, null, null)
+            val replies1To2 = testBuilder.systemAdmin.replies.listReplies(metaform.id, null, null, null, null, null, null, null, 0, 2, null, null)
```

Comment:
This test no longer tests whether null firstResult with maxResults work, is this change intentional?

Reply:
Unintentional change, reverted